packages/documentation/copy/en/handbook-v2/Type Manipulation/Conditional Types.md

```diff
@@ -38,6 +38,46 @@ type Stuff =
 
 When the type on the left of the `extends` is assignable to the one on the right, then you'll get the type in the first branch (the "true" branch); otherwise you'll get the type in the latter branch (the "false" branch).
 
+> Be aware that this is not true for union types.
```

Comment:
I missed that, since I read this document until I found something that's incorrect and contradicts the examples it uses. Having a section people don't know they need which contradicts statements in the introduction is a good way to leave people like me confidently misinformed after skimming the top of the document.

To mitigate people in the future making the same mistake, I think a note and a link to that section should be included here (and possible in the example where it depends on that behavior), then I can integrate my examples into that section.

Does that seem like a good approach?

Reply:
I don't like interrupting the flow of the Handbook with caveats that you need to read the entire page you're currently reading; the entire Handbook is intended to be consumed linearly and all sections include generalizations that are later fleshed out in more detail.